db/migrate/20160429135101_add_attributes_to_course_discussion_topics.rb

```diff
@@ -0,0 +1,8 @@
+class AddAttributesToCourseDiscussionTopics < ActiveRecord::Migration
+  def change
+    change_table :course_discussion_topics do |t|
+      t.references :course, null: false
```

Comment:
This breaks migrations as there's no sensible default value for `course_id` and the column cannot be null. A db:reset is necessary.

When there's production data, a db reset is not feasible so null values have to be allowed first.